statuslight_control/test/statuslight_control/statuslight_test.exs

```diff
@@ -1,15 +1,14 @@
-defmodule StatuslightControl.StatuslightTest do
+defmodule StatuslightControl.StatuslightGenserverTest do
```

Comment:
I need to revert this name change, I had changed it trying to debug an issue I thought had to do with namespacing but it wasn't needed